go/consensus/tendermint/apps/governance/transactions.go

```diff
@@ -43,6 +44,11 @@ func (app *governanceApplication) submitProposal(
 		return fmt.Errorf("failed to fetch consensus parameters: %w", err)
 	}
 
+	// To not violate the consensus, change parameters proposals should be ignored when disabled.
+	if proposalContent.ChangeParameters != nil && !params.EnableChangeParametersProposal {
+		return governance.ErrInvalidArgument
```

Comment:
I'm aware that different errors are returned, just wasn't sure if it is breaking. Higher in the stack trace we return `module`, `code` and `message`, so I thought all are breaking or none.
```
Codespace: module,
Code:      code,
Log:       err.Error(),
```
Will fix and test with different versions.

Reply:
Yeah `Log` is not consensus breaking.

Reply:
Offline discussion revealed that the `module` (`codespace` in Tendermint) is currently not consensus breaking, only the `code` is. Since all unknown errors `errUnknownError` (`module=unknown, code=1`) have the same `code` as `ErrInvalidArgument` (`module=governance, code=1`) errors, current code "magically" works.
To avoid having different errors I changed `ExecuteTx` to return `ErrInvalidArgument` when the body of a transaction is invalid.